src/components/home/HomeView.css

```diff
@@ -33,12 +33,36 @@
 
 @media (max-width: 800px) {
   .HomeView .left {
-    width: 100% !important;
+    width: 100%;
   }
   .HomeView .modal {
-    display: inline-block !important;
+    display: inline-block;
   }
   .HomeView .right {
-    display: none !important;
+    display: none;
   }
 }
+
+.HomeView .right.mobile {
+  z-index: 2;
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 300px;
+}
+
+.HomeView .left .greybox {
+  display: none;
+}
+
+.HomeView .left .greybox.active {
+  position: absolute;
+  z-index: 1;
+  display: block;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: #80808080;
```

Comment:
I think you meant #808080

Reply:
Nah the last 80 is the transparency

Reply:
Wait I'm so shook I've never seen this before